binderhub/tests/test_build.py

```diff
@@ -17,17 +17,26 @@
 
 @pytest.mark.asyncio(timeout=900)
 @pytest.mark.parametrize("slug", [
-    "gh/binderhub-ci-repos/requirements/d687a7f9e6946ab01ef2baa7bd6d5b73c6e904fd",
-    "git/{}/d687a7f9e6946ab01ef2baa7bd6d5b73c6e904fd".format(
+    "gh/binderhub-ci-repos/requirements/20c4fe55a9b2c5011d228545e821b1c7b1723652",
+    "git/{}/20c4fe55a9b2c5011d228545e821b1c7b1723652".format(
         quote("https://github.com/binderhub-ci-repos/requirements", safe='')
     ),
     "git/{}/master".format(
         quote("https://github.com/binderhub-ci-repos/requirements", safe='')
     ),
-    "gl/minrk%2Fbinderhub-ci/0d4a217d40660efaa58761d8c6084e7cf5453cca",
 ])
 @pytest.mark.remote
 async def test_build(app, needs_build, needs_launch, always_build, slug, pytestconfig):
+    """
+    Test build an image a set of times against a repo which is very simple to
+    build.
+    
+    This test should only be run once when the build is actually taking place
+    without cache. If any commits are made to master of
```

Comment:
I don't really understand the first sentence (line 34 and 35). This test checks that several different ways of specifying a build slug work and get handled correctly.

Reply:
Yeah line 34 and 35's communication didn't turn out well. My intention was to communicate that while the test_build test case run several times with different arguments as input, it is meant to avoid making time consuming builds by being able to use caches of previous builds.

Reply:
Comment no longer around to cause confusion!